kubermatic-addons/custom-addon/ingress-nginx/deploy.yaml

```diff
@@ -274,11 +263,14 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    #ensure internal LB for nginx ingress
+    service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```

Comment:
If we add this annotation - we enforce internal load-balancer in Azure.. Would that be correct? Asking since this is community component and may get used in public deployments as well

Reply:
your right, we should remove this by default